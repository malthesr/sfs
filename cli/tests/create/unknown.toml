args = [
  "create",
  "-S",
  "tests/create/unknown.samples",
  "tests/create/simple.bcf"
]
status = "failed"
