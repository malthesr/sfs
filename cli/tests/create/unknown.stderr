unknown sample sample1000
